opentelemetry-sdk/src/logs/logger_provider.rs

```diff
@@ -160,7 +160,7 @@ impl LoggerProviderInner {
 impl Drop for LoggerProviderInner {
     fn drop(&mut self) {
         if !self.is_shutdown.load(Ordering::Relaxed) {
-            otel_info!(
+            otel_debug!(
```

Comment:
would like to discuss more on this one. My original intention behind keeping this info was:
If a user forgets to keep the LoggerProvider around and its gets dropped, then Logs will not flow. This log message would have helped figure out why logs are not flowing in that scenario.

Given majority users do the right thing (i.e they keep LoggerProvider active), this should not cause a noise, right? And we recommend people to explicitly invoke shutdown(), so the else branch of this code is going to be executed in normal flow, which only does debug level.

Reply:
Yeah, that makes sense to me. Changed both of these ones back